src/components/pages/templates/title-and-question.ts

```diff
@@ -0,0 +1,12 @@
+const titleAndQuestionTemplate = `
```

Comment:
Add a comment to these placeholder templates describing that they are essentially placeholders.